libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -218,7 +218,7 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 			m_unusedStackSlots.erase(m_unusedStackSlots.begin());
 			m_context->variableStackHeights[&var] = slot;
 			if (int heightDiff = variableHeightDiff(var, varName, true))
```

Comment:
It would be nice to make `variableHeightDiff` return `size_t`

But this would mean changing the type of `	std::map<Scope::Variable const*, int> variableStackHeights;` too

Reply:
Changed